packages/strapi-design-system/src/Combobox/ComboboxList.js

```diff
@@ -1,22 +0,0 @@
-import { useEffect } from 'react';
-import PropTypes from 'prop-types';
-
-import { maintainScrollVisibility } from './utils';
-
-export const ComboboxList = ({ options, activeOptionRef }) => {
-  useEffect(() => {
-    if (activeOptionRef.current) {
-      maintainScrollVisibility(activeOptionRef.current);
-    }
-  }, [activeOptionRef]);
-
-  return options;
-};
```

Comment:
ℹ️  I didn't really understand the need for this component other than to call the useEffect and maintainScrollVisibility when there is an activeOptionRef. Removing this resolves the bug and the component seems to function fine without but let me know if there is something I'm missing...

Reply:
Might have been a hangover from when we weren't using `floating-ui` 🤔